tee-worker/omni-executor/native-task-handler/src/types.rs

```diff
@@ -4,7 +4,7 @@ use parity_scale_codec::{Decode, Encode};
 use std::vec::Vec;
 
 #[derive(Encode, Decode, Debug, PartialEq, Eq)]
-pub enum NativeOperationOk {
+pub enum NativeTaskOk {
 	CallResponse(CallResponse),
 	QueryResponse(QueryResponse),
```

Comment:
Should we merge these as well? `NativeOperationOk` can use the variants of `CallResponse ` directly`. Wdyt?

Reply:
Ah yes - let me change that